smartapps/juano2310/jawbone-up-connect.src/jawbone-up-connect.groovy

```diff
@@ -28,16 +27,16 @@ mappings {
 	path("/receivedToken") { action: [ POST: "receivedToken", GET: "receivedToken"] }
 	path("/receiveToken") { action: [ POST: "receiveToken", GET: "receiveToken"] }
 	path("/hookCallback") { action: [ POST: "hookEventHandler", GET: "hookEventHandler"] }
+  path("/oauth/initialize") {action: [GET: "oauthInitUrl"]}
 	path("/oauth/callback") { action: [ GET: "callback" ] }
 }
 
-def getSmartThingsClientId() {
-   return appSettings.clientId
-}
-
-def getSmartThingsClientSecret() {
-   return appSettings.clientSecret
-}
+def getSmartThingsClientId() { return appSettings.clientId }
+def getSmartThingsClientSecret() { return appSettings.clientSecret }
+def getServerUrl() { return "https://graph.api.smartthings.com" }
+def getShardUrl() { return getApiServerUrl() }
```

Comment:
This is purely opinionated but since getShardUrl and getApiServerUrl are synonymous here I think we should just remove this method and access `apiServerUrl` everywhere.